Lib/test/test_typing.py

```diff
@@ -434,6 +434,10 @@ def test_repr(self):
         self.assertEqual(repr(Tuple[int, ...]), 'typing.Tuple[int, ...]')
         self.assertEqual(repr(Tuple[list[int]]), 'typing.Tuple[list[int]]')
 
+    def test_or_and_ror(self):
+        self.assertEqual(Tuple | int, Union[Tuple | int])
+        self.assertEqual(int | Tuple, Union[int, Tuple])
```

Comment:
This isn't actually testing that `Tuple` `__ror__` and `__or__` are working properly. `int` and builtin types provide their own `__or__` . Here's a counterexample:
```python
class Meta(type):
 def __or__(*args):
  return NotImplemented
 def __ror__(*args):
  return NotImplemented

# Doesn't support __or__
class X(metaclass=Meta): ...

# Works!
>>> X | int
__main__.X | int
# Boo!
>>> X | X
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unsupported operand type(s) for |: 'Meta' and 'Meta'
```
I suggest removing this, and instead seeing my next suggestion/comment below to test `_SpecialGenericAlias | _SpecialGenericAlias`

Reply:
After some more thought, I think we can remove this section, since we already test that code path with `Callable | Tuple`. Could you please remove it?